src/components/table/Table.vue

```diff
@@ -108,6 +108,23 @@
                             </template>
                         </th>
                     </tr>
+                    <tr v-if="searchable && hasSearchablenewColumns">
+                        <th
+                            v-for="(column, index) in newColumns"
+                            v-if="(column.visible || column.visible === undefined)"
+                            :key="index"
+                            :style="{ width: column.width + 'px' }">
+                            <div class="th-wrap">
+                                <div v-if="column.searchable">
+                                    <b-field>
+                                        <b-input
+                                            v-model="filters[column.field]"
+                                            :type="column.numeric ? 'number' : 'text'" />
+                                    </b-field>
+                                </div>
```

Comment:
We don't need all, you might change it to
```
<template v-if="column.searchable">
     <b-input
         v-model="filters[column.field]"
         :type="column.numeric ? 'number' : 'text'" />
</template>
```
And remember to add bInput into components attribute of bTable because it's important for a correct bundle

Reply:
I wants to full-text search in this [Table](https://imgur.com/a/5gasuPL)

Reply:
@viralpatel380 please wait, it will be merged after this review

Reply:
@jtommy - Just to confirm I understand your comment about adding bInput, that is just add 

import Input from '../input/Input'
...
...
components: {
       ...
       ...

        [Input.name]: Input,

Thanks for the thorough review!

Reply:
Correct!